src/box/box.cc

```diff
@@ -2540,6 +2540,12 @@ box_wait_quorum(uint32_t lead_id, int64_t target_lsn, int quorum,
 	return 0;
 }
 
+/**
+ * The pool is used by box_cc to allocate sync_trigger_data
+ * that is used in box_collect_confirmed_vclock.
```

Comment:
Please, describe why we need dynamic allocation. We use it in `box_collect_confirm_vclock` and in `on_relay_thread_start` trigger. They're executed in different fibers, so we cannot free data in `box_collect_confirm_vclock`

Reply:
Done.